packages/gatsby/src/utils/webpack.config.js

```diff
@@ -28,6 +28,12 @@ module.exports = async (
 ) => {
   const directoryPath = withBasePath(directory)
 
+  await fs.writeFile(
```

Comment:
Is the stage able to be passed through to the loaders as an argument? That seems like it would be easier.

Reply:
No idea... I'll just set it on `process.env` actually instead of this method.